tools/bootloader/usb_flash.c

```diff
@@ -583,8 +592,8 @@ void handler_erase(FirmwareErase *msg)
 
             /* Erase application section */
             for (int i = 7; i <= 11; ++i) {
-                bl_flash_erase_word(i);
                 layoutProgress("Preparing for upgrade", ((i + 3 - 7 + 2) * 1000 / 9));
+                bl_flash_erase_word(i);
```

Comment:
don't worry about this one... I'll get it in my PR.

Reply:
fixed